src/components/UI/Buttons/Buttons.tsx

```diff
@@ -1,19 +1,66 @@
 import { JSX } from 'preact';
 import styles from './Buttons.module.sass';
+import backIcon from './icons/backArrow.svg';
+import nextIcon from './icons/nextArrow.svg';
 
 interface ButtonProps extends JSX.HTMLAttributes<HTMLButtonElement> {
-  onClick: () => void;
-  children: JSX.Element | string;
+  onClick: (() => void) | undefined;
+  children: JSX.Element | JSX.Element[] | string;
 }
 
 /**
- * Button components for the UI
+ * Basic Button component
  */
+export function Button(props: ButtonProps) {
+  return <button className={styles.button} {...props}>{props.children}</button>;
+}
 
+/**
+ * Primary Button
+ */
 export function PrimaryButton(props: ButtonProps) {
-  return <button className={styles.primaryButton}>{props.children}</button>;
+  return (
+    <Button className={styles.primaryButton} onClick={props.onClick}>
+      {props.children}
+    </Button>
+  );
 }
 
+/**
+ * Secondary Button
+ */
 export function SecondaryButton(props: ButtonProps) {
-  return <button className={styles.secondaryButton}>{props.children}</button>;
+  return (
+    <Button className={styles.secondaryButton} onClick={props.onClick}>
```

Comment:
`onClick` it's part of `props`.
```suggestion
    <Button className={styles.secondaryButton} {...props}>
```